packages/pwa-buildpack/lib/cli/create-project.js

```diff
@@ -87,8 +87,6 @@ async function findTemplateDir(templateName) {
     }
 }
 
-module.exports.sampleBackends = require('../../sampleBackends.json');
```

Comment:
Not sure you should delete this just yet - it's still used here: https://github.com/magento/pwa-studio/blob/develop/packages/pwa-buildpack/lib/cli/generate-build-report.js#L12